### PR TITLE
Setup the workflow files.

### PR DESCRIPTION
**Pull request by CharliedeanCS**
I have created a workflow yaml file that will run a pylint and pytest on the pipeline folder.

This will ensure all pipeline files are testing and at a high quality.

The pylint score must be above 8.
All tests must pass.

### CODE REVIEW

File: pipeline/pipeline.py
Comment (Shurc1val) [inline pipeline/pipeline.py:1]: Why is this here with nothing in it?

Comment (CharliedeanCS) [reply pipeline/pipeline.py:1]: To ensure the initial workflow will run I had to setup a example main function and example test. Therefore all pull requests now to main will go through status checks

Comment (Shurc1val) [reply pipeline/pipeline.py:1]: Ahh, yes, makes sense

**Review by Shurc1val — APPROVED** [review]
Can't see any issues, suppose we'll know when we try it!
